pkg/services/authz/token_auth.go

```diff
@@ -25,7 +25,9 @@ func (t *tokenAuth) GetRequestMetadata(ctx context.Context, _ ...string) (map[st
 		return nil, err
 	}
 
-	return map[string]string{authn.DefaultAccessTokenMetadataKey: token.Token}, nil
+	const metadataKey = "X-Access-Token"
```

Comment:
This is a Nit and no need to argue about this, but I'm not convinced making the key private was the best move. :confused:

Reply:
I agree, I did not think we used it. We can fix that part